src/java.base/share/classes/sun/security/rsa/RSACore.java

```diff
@@ -137,6 +137,24 @@ public static int getByteLength(RSAKey key) {
         return rsa(msg, key, true);
     }
 
+    /*
+     * Checks whether the given key size can be verified using OpenSSL 1.0.x
+     * or will it throw an error
+     * OpenSSL 1.0.x cannot be used to verify keys with certain key sizes since
+     * it always pads the result
+     * The key sizes that will cause padding issues are the ones of size: 8n+1
+     * (where n is any whole number)
+     * Hence, this function returns false if the key size is can be expressed
+     * in the form of 8n+1
+     * 
+     * @param keySize The size of the key that needs to be checked
+     * @return        False if the key can be expressed as 8n+1, True otherwise
+     * See https://github.com/eclipse-openj9/openj9/issues/11500#issuecomment-833004636 for more info
+    */
```

Comment:
Please finish each sentence with a period. The final line of the comment should be indented one more space.

Reply:
Done.

Reply:
The comment doesn't reflect the actual use pattern. If the method returns `false`, OpenSSL 1.0.x is still used, but a different verification approach is used.